File: Assignment_33/Program3.c
#include<stdio.h>

#define TRUE 1
#define FALSE 0
typedef int BOOL;

BOOL ChkDigit(char *ch)
{
        if(*ch >= '0' && *ch <= '9')  
        {
            return TRUE;
        }
        else
        {
            return FALSE;
        }     
}
int main()
{
     char Arr[50] = {'\0'};
     BOOL bRet = FALSE; 

     printf("Enter Character :\n");
     scanf("%[^'\n']c",Arr);

     bRet = ChkDigit(Arr);

     if(bRet == TRUE)
     {
        printf("TURE\n");
     }
     else
     {
        printf("FALSE\n");
     }
     return 0;
}

File: Assignment_33/Program3.java
import java.util.*;

class ChkDigits
{
    public boolean isDigits(char ch)
    {
        if(ch >= '0' && ch <= '9')
        {
            return true;
        }
        else
        {
            return false;
        }   
    }     
}
public class Program3
{
    public static void main(String[] args)
    {
    
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter the Character:");
        char ch = sobj.next().charAt(0);

        ChkDigits cobj = new ChkDigits();
        boolean bRet = cobj.isDigits(ch);

        if(bRet)
        {
            System.out.println("TRUE");
        }
        else
        {
            System.out.println("FALSE");
        }

    }
}
